### PR TITLE
[frinx-machine] Add metrics settings and update chart images

### PR DESCRIPTION
**Pull request by JumasJM**
#### Checklist

- [ ] Title of the PR starts with chart name (e.g. `[krakend]`)
- [ ] Update the documentation if a new value has been added
- [ ] Update chart release notes *[annotation](https://artifacthub.io/docs/topics/annotations/helm/)*

### CODE REVIEW

File: charts/frinx-machine/values.yaml [hunk 1]
```diff
@@ -748,3 +631,8 @@ oauth2-proxy:
       provider = "github"
       login_url = "https://github.com/login/oauth/authorize"
       redirect_url = "https://fm.127.0.0.1.nip.io/oauth2/callback"
+
+  metrics:
+    enabled: *oauth2-enabled
+    serviceMonitor:
+      enabled: *oauth2-enabled
```
Comment (Jozefiel) [inline charts/frinx-machine/values.yaml:638]: *metrics-enabled ?

File: charts/frinx-machine/Chart.yaml [hunk 2]
```diff
@@ -7,56 +7,31 @@ description: |
   Further information is available on [docs.frinx.io](https://docs.frinx.io/frinx-machine/getting-started/).
 icon: https://avatars.githubusercontent.com/u/23452093?s=200&v=4
 type: application
-version: 9.0.1
+version: 9.1.0
```
Comment (Jozefiel) [inline charts/frinx-machine/Chart.yaml:10]: make it 10.0.0

File: charts/frinx-machine/values.yaml (hunk 1 above)
Comment (JumasJM) [reply charts/frinx-machine/values.yaml]: resolved

File: charts/frinx-machine/Chart.yaml (hunk 2 above)
Comment (JumasJM) [reply charts/frinx-machine/Chart.yaml]: changed

**Review by Jozefiel — APPROVED** [review]

**Pull request merged**